CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.3 (2024-05-04)
```

Comment:
Can you check if you branched from the correctly updated DD-groundwater_sign as this changelog is already part of the master/main branch